engines/decidim_hospitalet-surveys/app/commands/decidim_hospitalet/surveys/admin/create_survey_result.rb

```diff
@@ -89,6 +89,13 @@ def existing_user
           ).first
         end
 
+        def default_user
+          Decidim::User.where(
```

Comment:
@josepjaume I [suggested it before](https://github.com/HospitaletDeLlobregat/decidim-hospitalet/issues/124#issuecomment-281620881), but it implies modifying the code `decidim` and was turned down.